docker/build/cicd.centos.Dockerfile

```diff
@@ -20,11 +20,11 @@ ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH \
     BUILDCHANNEL=${BUILDCHANNEL} \
     DEFAULTNETWORK=${DEFAULTNETWORK} \
     FULLVERSION=${FULLVERSION} \
-    PKG_ROOT=${PKG_ROOT} \
-    GOPROXY=https://gocenter.io
```

Comment:
Why is GOPROXY removed?

Reply:
I'm not sure.  Perhaps over-zealousness :)

I'll restore it.